_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "SQL Workshop"
```

Comment:
Is this in a separate folder for SQL?

Reply:
The original path for _index.md was "sql_workshop > Intro" but after looking through some workshops, I noticed their _index.md files weren't in a separate folder inside their workshops so I changed the path from "sql_workshop > Intro" to "sql_workshop".

Reply:
Oh that makes sense. Feel free to ignore.